config/locales/mailings.en.yml

```diff
@@ -14,22 +14,22 @@ en:
         account_activation_instructions: Activate your account by clicking the button below
         account_activation_link: Activate your account for our member system by going to %{url}.
         activate_account: Activate account
-        activity_updates_html: 'Do you always want to stay informed about these activities? Join the %{facebook_group_link_start} Sticky-members Facebook group %{link_end} and like  %{facebook_page_link_start} our page %{link_end}! Aside from that, you can find all information that you could ever want on our website: %{sticky_site_link_start} svsticky.nl %{link_end}. You can also send a WhatsApp message to the Sticky Phone (+31617695025) saying "Hi + [your name], English" after saving it in your contacts. Then you’ll automatically be updated about future events!'
+        activity_updates_html: 'Do you always want to stay informed about these activities? Follow our %{instagram_page_link_start} Sticky-members Instagram %{link_end} and follow our %{linkedin_page_link_start} our page %{link_end}! Aside from that, you can find all information that you could ever want on our website: %{sticky_site_link_start} svsticky.nl %{link_end}. You can also join our %{whatsapp_promo_link_start} WhatsApp promotion channel %{link_end} to be notified about upcoming activities!'
```

Comment:
Shouldn't this be something like `Follow our %{instagram_page_link_start} Instagram %{link_end} and %{linkedin_page_link_start} LinkedIn %{link_end} for updates about Sticky!`?